core/src/main/scala/cats/Invariant.scala

```diff
@@ -176,10 +176,6 @@ object Invariant extends ScalaVersionSpecificInvariantInstances with InvariantIn
     cats.instances.ordering.catsContravariantMonoidalForOrdering
   implicit def catsContravariantMonoidalForPartialOrdering: ContravariantMonoidal[PartialOrdering] =
     cats.instances.partialOrdering.catsContravariantMonoidalForPartialOrdering
-  implicit def catsContravariantMonoidalForEq: ContravariantMonoidal[Eq] =
```

Comment:
I think this binary incompatible. Can we add the methods back, but remove the `implicit`? I think that is binary compatible.

Reply:
Yeah, let me take a look. That is likely possible.

Reply:
I opted for version `since 2.9.0` based on the tag I saw, let me know if another would be better.